cmsdb/processes/higgs.py

```diff
@@ -61,14 +526,40 @@
             "th_gaussian": 0.039,
         }),  # TODO: only preliminary
     },
+    aux={"production_mode_parent": h},
 )
 
+# Higgs decay channels
+h_ggf_htt = add_decay_process(h_ggf, h_decay_map.htt)
+h_ggf_hww = add_decay_process(h_ggf, h_decay_map.hww)
+h_ggf_hzz = add_decay_process(h_ggf, h_decay_map.hzz)
+h_ggf_hbb = add_decay_process(h_ggf, h_decay_map.hbb)
+h_ggf_hnonbb = add_decay_process(h_ggf, h_decay_map.hnonbb)
+h_ggf_hcc = add_decay_process(h_ggf, h_decay_map.hcc)
+h_ggf_hzg = add_decay_process(h_ggf, h_decay_map.hzg)
+h_ggf_hgg = add_decay_process(h_ggf, h_decay_map.hgg)
+h_ggf_hmm = add_decay_process(h_ggf, h_decay_map.hmm)
 
-h_ggf_tautau = h_ggf.add_process(
-    name="h_ggf_tautau",
-    id=11100,
-    xsecs=multiply_xsecs(h_ggf, const.br_h.tt),
-)
+# Higgs sub-decay channels
+# TODO: mapping of parent processes does not yet work here
```

Comment:
What is different here w.r.t., for instance, `h_ggf_htt` above?

Reply:
Oh I think the comment is just outdated. This has been solved by adding the `production_mode_parent` aux to each process.